static/usage/v8/theming/always-high-contrast-mode/react/main_tsx.md

```diff
@@ -22,7 +22,7 @@ import { personCircle, personCircleOutline, sunny, sunnyOutline } from 'ionicons
 function Example() {
   return (
     <>
-      <IonHeader class="ion-no-border">
+      <IonHeader>
```

Comment:
These demos likely always had the class, but I decided to remove them while I was in here. The iframe demo doesn't have the class, and the Stackblitz demos should match the iframe demos in terms of visual result